File: Difference of 2.py
def twos_difference(input): 
    input_s = sorted(input)
    result = []
    for i in range(len(input_s) - 1):
        for j in range(i + 1, len(input_s)):
            if input_s[j] - input_s[i] == 2:
                result.append((input_s[i], input_s[j]))
    return result

File: Difference of 2/Difference of 2.js
function twosDifference(input){  
 const input_s = [...input].sort((a,b) => a-b);
 let result = []
 for(let i=0;i<input_s.length-1;i++){
   for(let j=i+1;j<input_s.length;j++){
     if(input_s[j]-input_s[i]===2){
       result.push([input_s[i],input_s[j]])
     }
   }
 }
 return result
}
